packages/web-app-files/tests/unit/components/SideBar/Shares/FileLinks.spec.ts

```diff
@@ -87,12 +90,11 @@ describe('FileLinks', () => {
     })
 
     describe('when the add-new-link button is clicked', () => {
-      it('should call addNewLink', async () => {
-        const spyAddNewLink = jest.spyOn((FileLinks as any).methods, 'addNewLink')
-        const { wrapper } = getWrapper()
-        expect(spyAddNewLink).toHaveBeenCalledTimes(0)
+      // TODO: fix and add tests
```

Comment:
Is this still a todo ? 🙈

Reply:
Yes, unit tests will follow in a separate PR because of the strict release timeline.